tests/IceRpc.Protobuf.Tests/OperationTests.proto

```diff
@@ -15,6 +15,14 @@ service MyOperations {
   rpc serverStreamingOp(google.protobuf.Empty) returns (stream OutputMessage);
 
   rpc bidirectionalStreamingOp(stream InputMessage) returns (stream OutputMessage);
+
+  rpc idempotentOp(google.protobuf.Empty) returns(google.protobuf.Empty) {
```

Comment:
All the Protobuf tests and examples should use the standard Protobuf naming conventions. For RPC methods, it's Pascal case.